ci/jfrog-new-vpc-rt6-ec2.json

```diff
@@ -55,10 +55,6 @@
         "ParameterKey": "CertificateDomain",
         "ParameterValue": "localdomain"
     },
-    {
```

Comment:
@vinayagg Isn't this still needed?

Reply:
This was duplicate field, I removed this one but on line 75, SMLicensesName still exists